src/HearThis/Communication/AndroidSynchronization.cs

```diff
@@ -47,20 +48,40 @@ public static void DoAndroidSync(Project project)
 			dlg.ShowAndroidIpAddress(); // AFTER we set our IP address, which may be used to provide a default
 			dlg.GotSync += (o, args) =>
 			{
-				var theirLink = new AndroidLink();
-				// Enhance: some way to validate that we really got an IP address.
-				theirLink.AndroidAddress = AndroidSyncDialog.AndroidIpAddress;
-				var ourLink = new WindowsLink(Program.ApplicationDataBaseFolder);
-				var merger = new RepoMerger(project, ourLink, theirLink);
-				merger.Merge(dlg.ProgressBox);
-				//Update info.txt on Android
-				var infoFilePath = project.GetProjectRecordingStatusInfoFilePath();
-				File.WriteAllText(infoFilePath, project.GetProjectRecordingStatusInfoFileContent());
-				var theirInfoTxtPath = project.Name + "/" + Project.InfoTxtFileName;
-				theirLink.PutFile(theirInfoTxtPath, File.ReadAllBytes(infoFilePath));
-				theirLink.SendNotification("syncCompleted");
-				dlg.ProgressBox.WriteMessage("Sync completed successfully");
-				//dlg.Close();
+				try
+				{
+					var theirLink = new AndroidLink();
+					// Enhance: some way to validate that we really got an IP address.
+					theirLink.AndroidAddress = AndroidSyncDialog.AndroidIpAddress;
```

Comment:
Looks like this "Enhance" comment is actually dealt with in the _syncButton_Click method. Or is there something more that could/should be done here?